network-config-analyzer/NetworkConfigQuery.py

```diff
@@ -1006,10 +1031,11 @@ def exec(self, only_captured):
             for peer2 in peers_to_compare if peer1 in captured_pods else captured_pods:
                 if peer1 == peer2:
                     continue
-                captured1, conns1, _ = self.config1.allowed_connections(peer1, peer2)
-                if only_captured and not captured1:
+                conns1_all, captured1_flag, conns1_captured, _ = self.config1.allowed_connections(peer1, peer2)
+                if only_captured and not captured1_flag:
                     continue
-                _, conns2, _ = self.config2.allowed_connections(peer1, peer2)
+                conns1 = conns1_captured if only_captured else conns1_all
+                conns2, _, _, _, = self.config2.allowed_connections(peer1, peer2)
```

Comment:
```suggestion
                conns2, _, _, _ = self.config2.allowed_connections(peer1, peer2)
```
A redundant comma (not sure what's the effect it has)